jx_sqlite/query_table.py

```diff
@@ -110,9 +110,19 @@ def query(self, query):
         else:
             create_table = ""
 
-        if query.groupby:
+        if query.groupby and query.format != "cube":
+            query = QueryOp.wrap(query, schema)            
             op, index_to_columns = self._groupby_op(query, frum)
             command = create_table + op
+        elif query.groupby:
+            tmp = query.edges
+            query.edges = query.groupby
+            query.groupby = tmp
```

Comment:
you can swap values using `query.edges, query.groupby = query.groupby, query.edges`